modules/ROOT/pages/getting-started-proxy.adoc

```diff
@@ -48,42 +52,49 @@ then API Manager displays APIs to choose from. Select the API one that was recen
 +
 - *Managing Type*: You may choose between `Basic Endpoint` or `Endpoint with Proxy`. Here we are using the endpoint with proxy.
 +
-- *Implementation URI*: Enter the URL where the API implementation resides. For this example we are using http://my-implementation.company.com
+- *Mule version:* If you are managing an API for Mule 4 or later, select this checkbox. 
++
+In this example, we are creating the API for Mule 4.
++
+- *Implementation URI*: Enter the URL where the API implementation resides. 
 +
-If you are managing an API for Mule 4 or above, please make sure you select the checkbox. In this example, we are creating the API for Mule 4.
+For this example we are using http://my-implementation.company.com:
++
+image::getting-started-proxy-editing-api.png[editing-api,width=75%]
 +
 ----
-If you need further configuration, you may want to click in Advanced Options, and fill in the options available.
+For additional configuration options, click in *Advanced options*, and fill in the details.
 ----
 +
 . Click on `Save`.
 
-After that, the API settings view is shown with the current information of the recently imported API.
+The API settings view is displayed with the details of the imported API. Your API instance is now created:
 
 image::getting-started-proxy-api-settings.png[created-api-in-admin-dashboard]
 
-So far, we have created an instance of an API Specification that can now be managed.
+[[configure_and_deploy_to_cloudhub]]
+== Configure a Proxy and Deploy to Cloudhub
 
-== Configure and Deploy a Proxy to Cloudhub
+Deploying an API proxy to CloudHub enables you to protect your API implementation without having to worry about the infrastructure details, such as firewalls, and so on. 
 
-Deploying an API Proxy to CloudHub lets you quickly protect your API implementation without having to worry about the
-infrastructure details. We do all the hard work for you, so now let's do this final step:
+To configure and deploy the proxy:
 
-. In the API Settings view, make sure your `Proxy deployment target` configuration is set to `CloudHub`
-. Scroll down until you hit `Deployment Configuration`
+. In the API Settings view of API Manager, ensure that your `Proxy deployment target` configuration is set to `CloudHub`.
+//*where is this displayed*--Fede
+. Scroll down to `Deployment Configuration`:
 +
 image::getting-started-proxy-deployment-config.png[]
 +
-. In `Runtime Version`, select the runtime version you need, and set a name for your application
+. From `Runtime Version`, select the runtime version you need, and set a name for your application
 . Finally, click on `deploy`.
 
-Once you see the following pop-up, then your API proxy is successfully running in a CloudHub instance.
+The following message is displayed: 
 
 image::getting-started-proxy-success-deployment.png[width=66%]
 
-Congratulations! You’ve just set up your first managed API.
+Your API proxy is now successfully running in a CloudHub instance.
```

Comment:
And your API instance is ready to be managed?

Reply:
Left it. too much in one go.